src/core/main.c

```diff
@@ -517,7 +517,6 @@ static int parse_proc_cmdline_item(const char *key, const char *value, void *dat
         } else if (!value) {
                 const char *target;
 
-                /* SysV compatibility */
                 target = runlevel_to_target(key);
```

Comment:
why drop thi scomment? the support for acepting sysv runlevel names on the kernel cmdline is sysv compat, no? i mean, maybe it's not controlled by any ifdeffery, but it's still something we inherit from sysv and provide for compat.

Reply:
I don't someone to see the comment and remove the code by mistake. I'll reword the comment to make that clear.